types/node/ts3.1/process.d.ts

```diff
@@ -259,6 +261,12 @@ declare module "process" {
                     tls_ocsp: boolean;
                     tls: boolean;
                 };
+                /**
+                 * @deprecated since v14.0.0 - Calling process.umask() with no argument causes
+                 * the process-wide umask to be written twice. This introduces a race condition between threads,
+                 * and is a potential security vulnerability. There is no safe, cross-platform alternative API.
+                 */
+                umask(): number;
```

Comment:
Why add it if it is deprecated? Seem noone missed it till now.
Or do you have use for it?

Reply:
From my understanding, marking as deprecated is preferable to removing working code.

Reply:
I'm not talking about removing something. If I look at the diff `umask(): number` was not there before.